services/livesync-service-base.ts

```diff
@@ -7,6 +7,7 @@ import * as shell from "shelljs";
 import * as path from "path";
 import * as temp from "temp";
 import * as minimatch from "minimatch";
+import * as constants from "../../constants";
```

Comment:
you can move the constant that you need in mobile/constants.ts file of common lib :)